certbot/certbot/_internal/main.py

```diff
@@ -495,29 +484,69 @@ def _report_new_cert(config, cert_path, fullchain_path, key_path=None):
 
     """
     if config.dry_run:
-        _report_successful_dry_run(config)
+        display_util.notify("The dry run was successful.")
+        return
+
+    assert cert_path and fullchain_path, "No certificates saved to report."
+
+    display_util.notify(
+        ("\nSuccessfully received certificate.\n"
+        "Certificate is saved at: {cert_path}\n{key_msg}"
+        "This certificate expires on {expiry}.\n"
+        "These files will be updated when the certificate renews.\n{renew_msg}\n").format(
+            cert_path=fullchain_path,
+            expiry=crypto_util.notAfter(cert_path).date(),
+            key_msg="Key is saved at:         {}\n".format(key_path) if key_path else "",
+            renew_msg="Certbot will automatically renew this certificate in the background."
+                      if config.preconfigured_renewal else
+                      (f'Run "{cli.cli_constants.cli_command} renew" to renew '
+                       "expiring certificates. "
+                       "We recommend setting up a scheduled task for renewal, see "
```

Comment:
```suggestion
                       "We recommend setting up a scheduled task for renewal; see "
```

tiiiiiniest of nits, but otherwise, love it!